system/ee/ExpressionEngine/Controller/Fields/Fields.php

```diff
@@ -865,6 +865,16 @@ private function form(ChannelField $field = null)
             }
 
             $field_options = $dummy_field->getSettingsForm();
+            // When fieldtype settings contain fields with their own group toggles
+            // we need to loop through them and append the fieldtype group name
+            foreach ($field_options as &$option) {
+                foreach ($option['settings'] ?? [] as $key => $setting) {
```

Comment:
EE 6.x still supports PHP 5.6, and `??` operator was added in 7.0, so this code will cause issues